---
format:
  revealjs:
    css: styles.scss
    progress: false
    scrollable: true
    slide-number: false
    show-slide-number: all
    font-size: 12
mainfont: lato
logo: illustrations/logo.svg
---


## Agenda: 

1. Biological Question
2. Analysis
3. Concluding Remarks

## Biological Question: 

![](illustrations/prostate.svg)

## Androgen receptor: 

![](https://encyclopedia.pub/media/item_content/202212/639a757a2a326biomolecules-12-01768-g002.png)

## The question: 

Does different drugs targeting the AR have different downstream effects? 

## Background: 

::::{.columns}
:::{.column width="33%"}
![](figures_presentation/paper.png)
:::
:::{.column width="66%"}
- **Cell Line:** LNCaP (androgen-responsive prostate cancer)
- **Treatments:** 3 anti-androgen drugs (Bicalutamide, Enzalutamide, Apalutamide) + Control
- **Conditions:** Â±DHT (dihydrotestosterone)
- **Replicates:** n=2 biological replicates
- **Assay:** Bulk RNA-seq
- **Analysis:** Differential expression + pathway enrichment
:::
::::

## The experiment: 

:::{.columns}
::::{.column width="33%"}

Bicalutamide

\

![](BIC.png)

::::

::::{.column width="33%"}

Apalutamide

\

![](ARN.png)

::::

::::{.column width="33%"}

Enzalutamide

\

![](ENZ.png)

::::
:::

## The experiment: 

- Test the three drugs on prostate cancer cell lines. 
- Only 2 biological replicates
- Lack of statistical power


## Analysis plan: 

1. DE-genes and pathway enrichment
2. The Random forrest approach 
3. Networks by proximity - LIONESS (one-sample network construction)
4. Community analysis
5. PPI
6. Checking with Reactome
7. Assessing Network effects using CoLoMoTo 

## We dump some data: 

![](figures_presentation/Screenshot%202025-12-12%20at%2012.21.59.png)

## DE-genes using DESeq-2: 

![](figures_presentation/Screenshot%202025-12-12%20at%2012.21.41.png)

## Wikipathways DE genes: 

![](figures_presentation/Screenshot%202025-12-12%20at%2012.21.45.png)

## Summary of the DE-genes: 

![](figures_presentation/Screenshot%202025-12-12%20at%2012.22.06.png)

## DE-genes using random forest: 

![](figures_presentation/Screenshot%202025-12-12%20at%2012.23.22.png)

## Comparing the two methods: 

![](figures_presentation/Screenshot%202025-12-12%20at%2012.23.45.png)

## Community analysis shows the same: 

![](figures_presentation/Screenshot%202025-12-12%20at%2012.23.30.png)

![](networkCOmparison.png)

## What it shows: 

- Many differences.
- To keep Boolean model simple we focus on the ones that are shared

## Only 4 gene overlap: 

![](figures_presentation/Screenshot%202025-12-12%20at%2012.23.52.png)

## Running a boolean model: 

![](figures_presentation/arnau.png)

## Running a boolean model: 

![](figures_presentation/Screenshot%202025-12-12%20at%2012.24.02.png)

## Running base model: 

![](BNN.png)

## And the values of Our genes of interest

![](GenesInterest.png)

## SSH = 0

![](SSH_img.png)

## Comparing:

![](panelBoolNet.png)

## Next steps: 

